pkg/apis/planetscale/v2/vitesscluster_defaults.go

```diff
@@ -108,3 +109,41 @@ func defaultClusterBackup(backup *ClusterBackupSpec) {
 		backup.Engine = defaultBackupEngine
 	}
 }
+
+func defaultTopoReconcileConfig(vt *VitessCluster) {
```

Comment:
Pass in `*TopoReconcileConfig` since that's the smallest scope at which the function can do its job.

Reply:
Done https://github.com/planetscale/vitess-operator/pull/26/commits/b026aec4a4017c54b1651342e80dc811ff55c00c.